pkg/local_object_storage/engine/shards.go

```diff
@@ -276,7 +268,7 @@ func (e *StorageEngine) HandleNewEpoch(epoch uint64) {
 }
 
 func (s hashedShard) Hash() uint64 {
-	return hrw.Hash(
+	return murmur3.Sum64(
```

Comment:
@roman-khimov, so what do you think about it? is it OK to have `hrw` lib without `Hash` func?

Reply:
I've seen that. You can wrap murmur3 into `Hash`, but you can use just about any other hash as well. Providing some default can be useful.

Reply:
Will you update it to the new `hrw` with `Hash()`?

Reply:
sure. done